h/util/metrics.py

```diff
@@ -36,7 +36,7 @@ def record_search_query_params(params, separate_replies):
     ]
     # The New Relic Query Language does not permit _ at the begining
     # and offset is a reserved key word.
-    params = [("es_{}".format(k), params[k]) for k in keys if k in params]
+    params = [(f"es_{k}", params[k]) for k in keys if k in params]
```

Comment:
I'm surprised this isn't a violation for the short name. I suppose we've exempted `k`?
```suggestion
    params = [(f"es_{key}", params[key]) for key in keys if key in params]
```